.github/ISSUE_TEMPLATE/Bug_report.md

```diff
@@ -1,40 +1,62 @@
 ---
 name: Bug report
-about: Create a report to help us improve
+about: Report a bug with the WordPress block editor or Gutenberg plugin
 
 ---
 
-**Describe the bug**
-A clear and concise description of what the bug is.
+<!--
+Please fill out ALL sections. Bug reports with missing information will be
+closed.
 
-**To reproduce**
-Steps to reproduce the behavior:
+Before submitting a bug report:
+
+- Check if the bug has already been fixed by updating WordPress and/or Gutenberg.
+- Check if the bug is caused by a plugin by deactivating all plugins except Gutenberg.
+- Check if the bug is caused by a theme by activating a default theme e.g. Twenty Twenty.
+
+If this is a security issue, please report it in HackerOne instead:
+https://hackerone.com/wordpress
+-->
+
+**Description**
```

Comment:
I need to follow-up by changing these to proper headings, it's been bugging me for ages that they're not.

Reply:
It would be more accessible too. Want to push up a commit which does that? 🙂

Reply:
Done